packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -3145,35 +3153,27 @@ class StackViewport extends Viewport {
    * Assumes that the slice index is correct for this viewport
    */
   public setViewReference(viewRef: ViewReference): void {
-    if (!viewRef) {
+    if (!viewRef?.referencedImageId) {
```

Comment:
so seems like this is not (and was not) working with just `sliceIndex` can we handle that case too so that we are able to say `setViewReference({sliceIndex})`